.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
You shouldn't be committing anything in the .idea folder as that is related to your IDE. usually a git ignore file will ignore these files from your commit, but i guess this repo doesn't have that setup so you need to manually remove them and be careful not to add them to your commits. Also delete the .DS_Store file.

Reply:
yep, IDE/OS specific files should be ignored globally for all repos using a global gitignore file: https://help.github.com/en/github/using-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer